docs/reference/edot-sdks/java/troubleshooting.md

```diff
@@ -43,14 +43,44 @@ Determine if the issue is related to the agent by following these steps:
 1. Start the application with no agent and see if the issue is not present. Observe if the issue is again present when restarting with the agent.
 2. Check end-to-end connectivity without the agent by running one or more of the example apps in [elastic-otel-java](https://github.com/elastic/elastic-otel-java/blob/main/examples/troubleshooting/README.md). These use the OpenTelemetry SDK rather than the auto-instrumentation. They can confirm that the issue is specific to the Java agent or can otherwise identify that the issue is caused by something else.
 
-## Agent debug output
+## Agent debug logging
 
-Turn on debug output with `-Dotel.javaagent.debug=true` or by setting the `OTEL_JAVAAGENT_DEBUG` environment variable to `true`. Turning debug output on might add noticeable overhead. A lower overhead option is to set the `otel.traces.exporter` JVM property or the `OTEL_TRACES_EXPORTER` environment variable to the value `otlp,logging-otlp`: this configures the agent to send the traces both to the Collector and to also print it to the standard output in JSON format.
+Debugging output is verbose and might produce noticeable overhead on the application, there are two main strategies to use:
+- in case of a technical issue or exception with the agent, use [agent debugging](#agent-debugging)
+- if you need details on the captured data, use [per-signal debugging](#per-signal-debugging)
```

Comment:
```suggestion
As debugging output is verbose and might produce noticeable overhead on the application, follow one of these strategies when you need logging:

- In case of a technical issue or exception with the agent, use [agent debugging](#agent-debugging).
- If you need details on the captured data, use [per-signal debugging](#per-signal-debugging).
```